### PR TITLE
Amrvis doc revisions

### PR DESCRIPTION
**Pull request by etpalmer63**
## Summary
Revised wording and presentation of Amrvis documentation. Added a comment about installation directory locations.

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [ ] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [x] include documentation in the code and/or rst files, if appropriate

### CODE REVIEW

File: Docs/sphinx_documentation/source/Visualization.rst [hunk 1]
```diff
@@ -79,16 +102,20 @@ you can build into a pdf using pdflatex).
...
+   pictures in several different formats under ``File/Export``.  In 2D you can
+   right or center click to get line-out plots.  In 3D you can right or
+   center click to change the planes, and hold shift+(right or center)
    click to get line-out plots.
 
-   We have created a number of routines to convert AMReX plotfile data other
+   |
+
+   We have created a number of routines to convert AMReX plotfile data to other
    formats (such as matlab), but in order to properly interpret the
```
Comment (kngott) [inline Docs/sphinx_documentation/source/Visualization.rst]: Capitalize Matlab correctly? (Not that is was before :) )

File: Docs/sphinx_documentation/source/Visualization.rst [hunk 2]
```diff
@@ -7,68 +7,91 @@ Amrvis
...
-allows you to actually view the numbers in a spreadsheet that is nested to
-reflect the AMR hierarchy -- this can be handy for debugging. You can modify how
-many levels of data you want to see, whether you want to see the grid boxes or
-not, what palette you use, etc. Below are some instructions and tips for using
-Amrvis; you can find additional information in Amrvis/Docs/Amrvis.tex (which
-you can build into a pdf using pdflatex).
+the ``amrvis1d``, ``amrvis2d``, and ``amrvis3d`` executables, and use
+them to visualize your data. A useful feature is ``View/Dataset``, which
+allows you to view data in a nested spreadsheet that
+reflects the AMR hierarchy -- this can be handy for debugging. In Amrvis you can modify how
```
Comment (kngott) [inline Docs/sphinx_documentation/source/Visualization.rst]: In Amrvis you

Add a comma?

File: Docs/sphinx_documentation/source/Visualization.rst [hunk 3]
```diff
@@ -7,68 +7,91 @@ Amrvis
...
+       alias amrvis2d /tmp/Amrvis/amrvis2d.<ver>.ex
+
+   |
+
+#. **Configure**:
+
+   Amrvis will look for the configuration file ``.amrvis.defaults`` in your
+   home folder. Run the command ``cp Amrvis/amrvis.defaults ~/.amrvis.defaults`` to
+   copy it there.  Then, in
+   your copy, edit the line containing "palette" to point to the color palette
```
Comment (kngott) [inline Docs/sphinx_documentation/source/Visualization.rst]: Should we explicitly state here that there are initial .amrvis.defaults and palette files in the directory? Seems to have been a little confusion about that recently.

File: Docs/sphinx_documentation/source/Visualization.rst (hunk 1 above)
Comment (etpalmer63) [reply Docs/sphinx_documentation/source/Visualization.rst]: Yup :+1:

File: Docs/sphinx_documentation/source/Visualization.rst (hunk 3 above)
Comment (etpalmer63) [reply Docs/sphinx_documentation/source/Visualization.rst]: I rewrote this paragraph to make those things more explicit. Please take a look at the new version.

File: Docs/sphinx_documentation/source/Visualization.rst (hunk 2 above)
Comment (etpalmer63) [reply Docs/sphinx_documentation/source/Visualization.rst]: Decided to replace the whole sentence.

**Review by kngott — APPROVED** [review]

**Pull request merged**